miso-web/src/main/webapp/scripts/list_servicerecord.js

```diff
@@ -114,7 +114,7 @@ ListTarget.servicerecord = {
             list += "</ul>";
             return list;
           }
-          return data;
+          return null;
```

Comment:
Why `null` instead of something like a count of how many files there are?

Reply:
From my understanding, null will only be returned when there is no file attachment to the service records. If there is an attachment, the list of hyperlinks to each file attachment will be returned. My intention was to make columns with no file attachment to be empty.

Reply:
Will do!